subworkflows/local/phylogenetic_qc.nf

```diff
@@ -0,0 +1,49 @@
+//
+// Seqinspector Phylogenetic classification of reads, to check for contamination and adjacent issues
+//
+
+include { UNTAR as UNTAR_KRAKEN2_DB } from '../../modules/nf-core/untar/main.nf'
+include { KRAKEN2_KRAKEN2 } from '../../modules/nf-core/kraken2/kraken2/main.nf'
+include { KRONA_KTUPDATETAXONOMY } from '../../modules/nf-core/krona/ktupdatetaxonomy/main.nf'
+include { KRONA_KTIMPORTTAXONOMY } from '../../modules/nf-core/krona/ktimporttaxonomy/main.nf'
+
+workflow PHYLOGENETIC_QC{
+    take:
+    reads
+
+    main:
+    ch_reads = reads
+    ch_multiqc_files = Channel.empty()
+    ch_versions             = Channel.empty()
+    //
+    // MODULE: Untar kraken2_db
+    //
+    UNTAR_KRAKEN2_DB ( [ [:], params.kraken2_db ])
+    ch_kraken2_db = UNTAR_KRAKEN2_DB.out.untar.map { it[1] }
+
+    //
+    // MODULE: Perform kraken2
+    //
+    KRAKEN2_KRAKEN2 (
+        ch_reads,
+        ch_kraken2_db,
+        params.kraken2_save_reads,
+        params.kraken2_save_readclassifications
+    )
+    ch_multiqc_files       = ch_multiqc_files.mix( KRAKEN2_KRAKEN2.out.report )
+    ch_versions            = ch_versions.mix( KRAKEN2_KRAKEN2.out.versions.first() )
+    //KRAKEN2_KRAKEN2.out.report.map { meta, report -> [ report ] }.collect()
+
+    //
+    // MODULE: krona plot the kraken2 reports
+    //
+    KRONA_KTUPDATETAXONOMY()
+    KRONA_KTIMPORTTAXONOMY (
+        KRAKEN2_KRAKEN2.out.report,
+        KRONA_KTUPDATETAXONOMY.out.db
+    )
```

Comment:
If Krona's output is not added to the `emit:` section, it won't appear in the pipeline output files.

I'm not familiar with Krona, are these plots supposed to be added to the multiqc report or is it supposed to be its own report?

Reply:
The [krona plots](https://github.com/marbl/Krona/wiki) are an interactive alternative to MultiQC to visualize the kraken2 results. I've added the emit now!